src/main/java/org/folio/clients/ClientUtilAsync.java

```diff
@@ -0,0 +1,56 @@
+package org.folio.clients;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
```

Comment:
Wasn't this branch taken from master? This seems to be part of older branch?

Reply:
Yes, it this a problem? i can create new one if necessary. Only last commit needed review here

Reply:
Sure, I thought the other changes are already in master.